### ISSUE TITLE
Security issue I found online

### ISSUE THREAD

**Issue by TamkeenLMS** [evolutionscript/HelpDeskZ-1.0#45]
Hi there. Thanks for the awesome script. 

While I was doing a search about HelpDeskz I found this [link](https://packetstormsecurity.com/files/138548/HelpDeskz-1.0.2-Shell-Upload.html). Please take a look at it and fix what should be fixed. 

Thanks again.

**Comment by turbo2ltr** [comment]
Wow, sucks this package doesn't seem to be maintained anymore.

**Comment by andysavage** [comment]
Yes it's a shame... this is the nicest-looking help desk thing there is. All the others look quite complicated and off-putting to users. I too hope someone will adopt it and fix issues and further the dev.

**Comment by turbo2ltr** [comment]
After finding out that this script is not being maintained, I decided to try out osTicket.  Definitely doesn't look as good and has a rather confusing UI. And the documentation is completely non-existent.

**Comment by cboyce428** [comment]
Just a FYI for anyone concerned with this strictly speaking this is a configuration error in apache, 

If you look at the "uploads" directory you can see it has "deny from all" within the .htaccess file, by default apache does not allow .htaccess files to be run unless "AllowOverride" configuration is changed from None to All (/etc/apache2/apache2.conf) with this changed this security risk is not possible.

Best practices are to not make the user uploaded files web accessible so it should really be amended.
